docs/admin/setup/distributed.rst

```diff
@@ -502,3 +502,34 @@ It relays message between instances:
 
    .. figure:: distributed/registry.png
       :align: center
+
+
+Running multiple instances locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For testing purposes, you can also run multiple instances on the same machine. This can be done by using
+different ports and a unique instance ID for each instance.
+
+#. In ``application-local.yml``, add the following configuration:
+
+  .. code:: yaml
+
+    eureka:
+        client:
+            enabled: true
```

Comment:
```suggestion
  .. code:: yaml

     eureka:
         client:
             enabled: true
```
Needs to be indented with one additional space to fix the item numbering.

Currently:
![image](https://github.com/ls1intum/Artemis/assets/64250573/73aec5a5-3398-475a-8f45-19923491e1a5)

Reply:
sorry about that. Now fixed